src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -85,6 +84,7 @@ const state = reactive({
     } as DateRange,
     isProjectDashboard: computed<boolean>(() => props.dashboardId.startsWith('project')),
     dashboardInfo: {} as DashboardModel,
+    dashboardName: '',
```

Comment:
Your and mine codes would cause conflict 🥲 
Merge your commit first, and I'll resolve it

Reply:
Thank you 🥹